test/service/test_manifest.py

```diff
@@ -2129,19 +2129,19 @@ def test_compact_manifest(self):
         'files.is_supplementary': {'is': [True, False]}
     }
 
+    orphan_cases = [
```

Comment:
When this is moved further away from the usage, its structure becomes a bit opaque and should be documented with a brief comment.